assets/js/base/components/totals/totals-item/index.js

```diff
@@ -24,8 +30,9 @@ const TotalsItem = ( { className, label, value, description } ) => {
 };
 
 TotalsItem.propTypes = {
+	currency: PropTypes.object.isRequired,
 	label: PropTypes.string.isRequired,
-	value: PropTypes.string.isRequired,
+	value: PropTypes.number.isRequired,
```

Comment:
Hmm, are we sure this is a number? I _think_ values coming from the REST response are always a string.